src/main/groovy/com/netflix/nebula/lint/plugin/LintGradleTask.groovy

```diff
@@ -18,15 +18,24 @@ package com.netflix.nebula.lint.plugin
 import com.netflix.nebula.lint.*
 import org.gradle.api.DefaultTask
 import org.gradle.api.GradleException
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.Optional
 import org.gradle.api.tasks.TaskAction
-import org.gradle.util.DeprecationLogger
 
 import static com.netflix.nebula.lint.StyledTextService.Styling.*
 
 class LintGradleTask extends DefaultTask {
+    @Input
+    @Optional
```

Comment:
Optional is a cool annotation! Nice! Are we using it here because it has a default value & so can be unspecified?

Reply:
basically it could be provided or not. IF not, we set a default but it doesn't require user to provide one always